SECURITY.md

```diff
@@ -0,0 +1,46 @@
+# Security Notice
+
+## What?
+
+This is the security notice for all GDS (Government Digital Service) `alphagov` repositories. The notice explains how vulnerabilities should be reported to GDS. At GDS there are cyber security and information assurance teams, as well as security-conscious people within the programmes, that assess and triage all reported vulnerabilities.
+
+## Reporting a Vulnerability
+
+GDS is an advocate of responsible vulnerability disclosure. If you’ve found a vulnerability, great! We would like to know so we can fix it.
+
+If you believe a vulnerability affects the data (release of sensitive data or personal identifiable information) of a system, send details to [disclosure@digital.cabinet-office.gov.uk], include:
+- the website, page or repository where the vulnerability can be observed
+- a brief description of the type of vulnerability and any related [OWASP category]
+- non-destructive exploitation details
+
+For all other vulnerabilities, you can leave an issue against the repository (if you are not sure about which repository to raise an issue against, you can still email details through to [disclosure@digital.cabinet-office.gov.uk]):
+- prefix the issue title with `[Vulnerability]`
+- add details to the issue about the impact and any related [OWASP category]
+- don’t include detailed exploitation steps publically, GDS may respond in the issue's comments or contact you via your GitHub profile (alternatively, you can leave other contact means)
+
+GDS use ZenDesk as a ticketing system, so you may get a reply or response from that platform.
```

Comment:
A lot of people are unfamiliar with ZenDesk - isn't it just an email to the end user?

Reply:
The email address goes to a ZenDesk queue, so I think it's worth mentioning that they might get a reply from that platform